report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -60,14 +62,16 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
         },
         '221': function () {
           scope.traverseTree('down');
-        }
+        },
+        '65': accept,
```

Comment:
Great job!
What do you think, maybe we could remember ASCII codes in variables instead of add comment?

Reply:
Actually we don't need to use ASCII codes at all and we can use event.key property instead of event.keyCode which is returning single character that identify keyboard buttons. I added new shortcuts using existing convention. If it is necessarily I can quick refactor it.